medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/PatientService.java

```diff
@@ -0,0 +1,64 @@
+package com.Mafiuz04.medicalclinic;
+
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PatientService {
+    List<Patient> patients = new ArrayList<>();
+
+    public void showAllPatients() {
+        for (Patient patient : patients) {
+            System.out.println(patient);
+        }
+    }
+
+    public void showPatientByEmail(String email) {
+        patients.stream()
+                .filter(patient -> patient.getEmail().equals(email))
+                .findAny()
+                .ifPresent(System.out::println);
+    }
+
+    public void addPatient(String email, String password, long idCardNo, String firstName, String lastName,
```

Comment:
Dlaczego dodając pacjenta nie przekażmy w parametrze po prostu Pacjenta, który ma zostać dodany?
Zauwaz, że jak będziesz miał endpoint jakis:

POST /patients to w body w requescie HTTP dostaniesz dane pacjenta, które masz dodać w formacie JSON 
{
"email" : "kg",
....

}

Takiego Jsona z BODY automatycznie w controllerze spring przekonwertuje nam na pacjenta, jesli typ w body podamy patient
W kontrolerze taka metoda: 
@PostMapping("/patients") 
public Patient addPatient(@RequestBody Patient patient) {
return patientService.addPatient(patient); <- brakuje wam całej warstawy @RestController odpowiedzialnej za komunikacje za swiatem zewnetrznym
}